jenkins/automation/alex.groovy

```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent {
           node {
-             label "docker-${os_version}-node"
+             label "ssc-vm-g4-rhev4-0465.colo.seagate.com"
```

Comment:
dont hardcode the hostname as it will be a security risk

Reply:
@mukul-seagate11 , Thanks for pointing out this. I have changed as a jenkins parameter now.